### PR TITLE
Mes 24

### PR DESCRIPTION
**Pull request by ppmarkek**

### CODE REVIEW

**Comment by ppmarkek** [comment]
Should be minimized on load

On the left icon, should popup sidebar

The title after the left icon should be taken from Page Title / ‘history page enum’ approach

On clicking the search icon - search input should appear

The profile picture should show a popup menu on click

File: src/components/templates/Novigation/Novigation.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                        </NovigButton>
+                    </ChangeButton>
+                    <Text variant={'H3'} small={''} color={''}>
+                        {location.pathname.replace('/', '')}
+                    </Text>
+                </Grid>
+                <Grid container xs={6} gap={'10px'} justifyContent={'flex-end'}>
+                    <Input
+                        variant={'Search'}
+                        type={''}
```
Comment (k0wapa) [inline src/components/templates/Novigation/Novigation.tsx:109]: These props might be optional

File: src/components/templates/Novigation/Novigation.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                        text={'Search'}
+                        width={''}
```
Comment (k0wapa) [inline src/components/templates/Novigation/Novigation.tsx:111]: Same for next 3 props, they should be optional

File: src/components/templates/Novigation/Novigation.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    const handleClick = (event: MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    return (
+        <Wrapper container>
+            <TopNovigation container xs={11.5}>
```
Comment (k0wapa) [inline src/components/templates/Novigation/Novigation.tsx:91]: TopNavigation

File: src/components/templates/Novigation/Novigation.tsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                        }}
+                        transformOrigin={{ horizontal: 'right', vertical: 'top' }}
+                        anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+                    >
+                        <MenuItem onClick={handleClose}>Settings</MenuItem>
+                        <MenuItem onClick={handleClose}>Logout</MenuItem>
+                    </Menu>
+                </Grid>
+            </TopNovigation>
+            <LeftNovigation container xs={0.5}>
```
Comment (k0wapa) [inline src/components/templates/Novigation/Novigation.tsx:159]: LeftNavigation

**Pull request merged**